src/main/java/hu/akarnokd/rxjava2/basetypes/BasicMergeSubscription.java

```diff
@@ -0,0 +1,335 @@
+package hu.akarnokd.rxjava2.basetypes;
```

Comment:
This doesn't belong there but in the `operators` subpackage. Please add the standard library header as well.